cmd/influxd/upgrade/config_test.go

```diff
@@ -372,7 +372,7 @@ influxql-max-select-buckets = 0
 influxql-max-select-point = 0
 influxql-max-select-series = 0
 log-level = "info"
-query-concurrency = 0
+query-concurrency = 9223372036854775807
```

Comment:
Should this use `launcher.MaxInt` like above or `math.MaxInt`?

Reply:
I tried it with `max.MaxInt` to start, and the test still passed with `0` here... assumed the number was too big and something funky was happening with the parser. I'll give it another try just to sanity-check.

Reply:
Looks like `launcher.MaxInt` == `math.MaxInt32`